src/TypeResolver.js

```diff
@@ -152,7 +152,7 @@ class TypeResolver {
         }
 
         if (key === 'bytes') {
-            return FateTypeBytes(valueTypes)
+            return FateTypeBytes(valueTypes === 'any' ? undefined : valueTypes)
```

Comment:
@dincho How do you like this? I would use `undefined` instead `'any'` because it works this way and `FateTypeBytes`, `FateBytes` are internal.

Reply:
I'm fine with undefined